packages/synthetix-main/contracts/token/SUSDToken.sol

```diff
@@ -20,7 +21,11 @@ contract SNXToken is ERC20, UUPSImplementation, Ownable, Authorizable {
         _upgradeTo(newImplementation);
     }
 
-    function mint(address recipient, uint256 amount) external onlyAuthorized {
-        _mint(recipient, amount);
+    function burn(address to, uint256 amount) external override onlyAuthorized {
```

Comment:
I know `onlyAuthorized` should be very secure but we might want to tighten this all the way down to `onlyOwner`. That will make it harder for an accident or attack somehow granting access to mint sUSD to an account outside of the core mvp system, and makes the governance process more clear.

Reply:
Fixed